components/TextImage.vue

```diff
@@ -1,12 +1,14 @@
 <template>
-  <div v-editable="blok">
-    <article>
-      <div>
-        <h2>{{ blok.title }}</h2>
-        <rich-text-renderer :document="blok.description" />
+  <div v-editable="blok"> 
+    <article class="container lg:flex">
+      <div class="w-full">
```

Comment:
de foto is niet 50% bij xl terwijl je wel dat aangeeft. je zou de parent van de tekst en titel een flex-1 geven dat neemt de rest van de space in de flexbox. omdat je nu 100% width geeft aan de div met tekst en titel en 50% aan de figure dus de parent div met de image. Hierdoor let de browser dat de tekst div 2 keer zo groot moet zijn dan de image dus werkelijk is het dus nu 66.6% tekst en 33.3% image